src/riak_kv_entropy_manager.erl

```diff
@@ -66,16 +70,20 @@
 -type vnode() :: {index(), node()}.
 -type exchange() :: {index(), index(), index_n()}.
 -type riak_core_ring() :: riak_core_ring:riak_core_ring().
+-type version() :: undefined | non_neg_integer().
 
 -record(state, {mode           = automatic :: automatic | manual,
                 trees          = []        :: [{index(), pid()}],
                 tree_queue     = []        :: [{index(), pid()}],
+                trees_version  = []        :: [{index(), version()}],
```

Comment:
Yep, agreed, will change and do the same with trees since the code always treats trees as an orddict as well.